tools/api_versioning/generate_api_version_header.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+"""Parses a file containing the API version (X.Y.Z format), and outputs (toxi
```

Comment:
toxi?

Reply:
Fixed (vim key strokes in insert mode...)